src/require-assert-message-rejects-throws.ts

```diff
@@ -0,0 +1,53 @@
+// require-assert-message-rejects-throws.ts
+
+/*
+ * Copyright (c) 2021-2024 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+import type { Rule } from 'eslint';
+
+export default {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Validate that error and message argument is always supplied to node:assert rejects,throws methods',
+      url: 'https://github.com/checkdigit/eslint-plugin',
+    },
+  },
+  create(context) {
+    return {
+      CallExpression(node) {
+        const callee = node.callee;
+        if (
+          callee.type === 'MemberExpression' &&
+          callee.object.type === 'Identifier' &&
+          callee.object.name === 'assert' &&
```

Comment:
i think its ok for now, but we are making an assumption that the import from node:assert will be called "assert", which may not be the case.